test/controllers/survey_controller_test.exs

```diff
@@ -2576,49 +2572,48 @@ defmodule Ask.SurveyControllerTest do
     end
 
     test "shows a panel survey", %{conn: conn, project: project} do
-      survey = insert(:survey, project: project)
-      survey = Survey.changeset(survey, %{panel_survey_of: survey.id}) |> Repo.update!()
+      panel_survey = insert(:panel_survey, project: project)
+      survey = insert(:survey, project: project, panel_survey: panel_survey)
 
       conn = get(conn, project_survey_path(conn, :show, project, survey))
 
-      assert json_response(conn, 200)["data"]["is_panel_survey"] == true
+      assert json_response(conn, 200)["data"]["panel_survey_id"] == panel_survey.id
     end
 
     test "shows a regular survey", %{conn: conn, project: project} do
       survey = insert(:survey, project: project)
 
       conn = get(conn, project_survey_path(conn, :show, project, survey))
 
-      assert json_response(conn, 200)["data"]["is_panel_survey"] == false
+      assert json_response(conn, 200)["data"]["panel_survey_id"] == nil
     end
 
-    test "sets up a panel survey", %{conn: conn, project: project} do
+    test "sets a panel survey", %{conn: conn, project: project} do
```

Comment:
we should't be able to to this right?

Reply:
I think we should allow doing it for now. Because currently, the UI first creates de survey, and then it's "converted" into a panel survey.

Reply:
yo are right!